### DISCUSSION TITLE
Animations not working

### DISCUSSION THREAD

**Discussion by amoghdhu** [community/community#61918] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey everyone! I just uploaded my local bootstrap portfolio onto Github Pages and none of the animating elements are showing up. I checked in console and I'm getting this error: Uncaught ReferenceError: AOS is not defined. If somone could assist on this matter, that would be greatly appreciated.

I attempted using solutions from [here](https://github.com/michalsnik/aos/issues/110) in regards to initializing AOS, but it didn't work.

**Comment by BaraaBasim** [comment]
Make sure that the AOS library is included in your HTML file. You can do this by checking the <head> section of your HTML file and confirming that the AOS stylesheet and JavaScript files are properly linked. For example:
```
<head>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/aos@2.3.4/dist/aos.css" />
  <script src="https://cdn.jsdelivr.net/npm/aos@2.3.4/dist/aos.js"></script>
</head>
```

Check that the AOS library is properly initialized in your JavaScript file. You should have a line of code that looks like this:
```
AOS.init();
```

If you have confirmed that the AOS library is properly included and initialized, try clearing your browser cache and reloading your GitHub Pages site. Sometimes, cached files can cause issues with loading scripts and stylesheets.

**Comment by fedorkanan** [comment]
hi dear Amogh :
what browser do you use for displaying your work 
you should see the wbkit property in every browser that doesn't support them like :
-webkit-
  -moz-
  -o-
  -ms-
before this property:
@keyframes
animation-name
animation-duration
animation-delay
animation-iteration-count
animation-direction
animation-timing-function
animation-fill-mode
animation
